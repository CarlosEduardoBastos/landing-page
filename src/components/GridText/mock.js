export default {
  background: false,
  title: 'My grid',
  description:
    'Lorem ipsum dolor, sit amet consectetur adipisicing elit. Sed, repellendus nobis? Id placeat, adipisci minus dicta, nisi quasi iste nostrum ratione voluptatem error repellat veritatis doloremque quaerat temporibus quidem accusantium.',
  grid: [
    {
      title: 'Teste 1',
      description:
        'Lorem ipsum dolor sit amet consectetur adipisicing elit. Consequuntur harum voluptas assumenda, dignissimos sapiente voluptatem amet, consequatur, inventore facere rerum tempore. Iusto veniam ut et odio. Sapiente exercitationem eum inventore.',
    },
    {
      title: 'Teste 2',
      description:
        'Lorem ipsum dolor sit amet consectetur adipisicing elit. Consequuntur harum voluptas assumenda, dignissimos sapiente voluptatem amet, consequatur, inventore facere rerum tempore. Iusto veniam ut et odio. Sapiente exercitationem eum inventore.',
    },
    {
      title: 'Teste 3',
      description:
        'Lorem ipsum dolor sit amet consectetur adipisicing elit. Consequuntur harum voluptas assumenda, dignissimos sapiente voluptatem amet, consequatur, inventore facere rerum tempore. Iusto veniam ut et odio. Sapiente exercitationem eum inventore.',
    },
  ],
};
