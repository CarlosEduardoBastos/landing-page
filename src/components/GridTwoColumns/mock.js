export default {
  title: 'Grid Two Columns',
  text: `Lorem ipsum dolor sit amet consectetur adipisicing elit. Ut, sed doloribus dignissimos sit maxime est quo alias quae in magni voluptatum natus dolor expedita laudantium similique voluptate nam aut enim!`,
  srcImg: 'assets/images/javascript.svg',
};
