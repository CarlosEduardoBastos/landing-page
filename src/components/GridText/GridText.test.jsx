import { GridText } from '.';
import { renderTheme } from '../../styles/render-theme';
import mock from './mock';

describe('<GridText />', () => {
  it('should render grid section', () => {
    const { container } = renderTheme(<GridText {...mock} />);
    expect(container).toMatchSnapshot();
  });
});
