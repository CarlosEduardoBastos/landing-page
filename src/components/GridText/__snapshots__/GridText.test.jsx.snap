// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<GridText /> should render grid section 1`] = `
.c4 {
  color: #0a1128;
  font-size: 6.4rem;
  text-transform: uppercase;
}

.c9 {
  color: #0a1128;
  font-size: 3.2rem;
  text-transform: none;
}

.c6 {
  font-size: 2.4rem;
}

.c2 .c5 {
  margin-bottom: 6.4rem;
}

.c7 {
  counter-reset: grid-counter;
  display: grid;
  grid-template-columns: repeat(auto-fill,minmax(280px,1fr));
  gap: 3.2rem;
  width: 100%;
  overflow: hidden;
}

.c8 .c3 {
  position: relative;
  left: 5rem;
}

.c8 .c3::before {
  counter-increment: grid-counter;
  content: counter(grid-counter);
  position: absolute;
  font-size: 7rem;
  top: -3rem;
  left: -5rem;
  -webkit-transform: rotate(5deg);
  -ms-transform: rotate(5deg);
  transform: rotate(5deg);
}

.c1 {
  max-width: 120rem;
  margin: 0 auto;
  padding: 3.2rem;
}

.c0 {
  background: #ffffff;
  color: #0a1128;
  min-height: 100vh;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

@media (max-width:768px) {
  .c4 {
    font-size: 4.0rem;
  }
}

@media (max-width:768px) {
  .c7 {
    grid-template-columns: 1fr;
  }
}

<div>
  <div
    class="c0"
  >
    <div
      class="c1"
    >
      <div
        class="c2"
      >
        <h2
          class="c3 c4"
        >
          My grid
        </h2>
        <div
          class="c5 c6"
        >
          Lorem ipsum dolor, sit amet consectetur adipisicing elit. Sed, repellendus nobis? Id placeat, adipisci minus dicta, nisi quasi iste nostrum ratione voluptatem error repellat veritatis doloremque quaerat temporibus quidem accusantium.
        </div>
        <div
          class="c7"
        >
          <div
            class="c8"
          >
            <h3
              class="c3 c9"
            >
              Teste 1
            </h3>
            <div
              class="c5 c6"
            >
              Lorem ipsum dolor sit amet consectetur adipisicing elit. Consequuntur harum voluptas assumenda, dignissimos sapiente voluptatem amet, consequatur, inventore facere rerum tempore. Iusto veniam ut et odio. Sapiente exercitationem eum inventore.
            </div>
          </div>
          <div
            class="c8"
          >
            <h3
              class="c3 c9"
            >
              Teste 2
            </h3>
            <div
              class="c5 c6"
            >
              Lorem ipsum dolor sit amet consectetur adipisicing elit. Consequuntur harum voluptas assumenda, dignissimos sapiente voluptatem amet, consequatur, inventore facere rerum tempore. Iusto veniam ut et odio. Sapiente exercitationem eum inventore.
            </div>
          </div>
          <div
            class="c8"
          >
            <h3
              class="c3 c9"
            >
              Teste 3
            </h3>
            <div
              class="c5 c6"
            >
              Lorem ipsum dolor sit amet consectetur adipisicing elit. Consequuntur harum voluptas assumenda, dignissimos sapiente voluptatem amet, consequatur, inventore facere rerum tempore. Iusto veniam ut et odio. Sapiente exercitationem eum inventore.
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
